### PR TITLE
Support multiple platforms for fonts

### PR DESCRIPTION
**Pull request by hedvigoscar**
- The names are different on Native and Web
- We use `react-primitives` to determine the names to export

Post merging, we need to:
- Publish the package

### CODE REVIEW

File: fonts.ts [hunk 1]
```diff
@@ -1,5 +1,19 @@
-export const fonts = {
-  MERRIWEATHER: "Merriweather-Light",
-  CIRCULAR: "CircularStd",
-  SORAY: "SoRay"
-};
+import {Platform} from 'react-primitives';
+
+let fonts;
+
+if (Platform.OS === 'web') {
```
Comment (Iamsamwhoami) [inline fonts.ts]: move this into a self invoking function and then we can avoid the let? :)

Comment (Iamsamwhoami) [reply fonts.ts]: also there is `Platform.select` you could use instead of an if statement

Comment (Iamsamwhoami) [reply fonts.ts]: so actually ```const fonts = Platform.select({ ... })```

File: fonts.ts [hunk 2]
```diff
@@ -1,5 +1,19 @@
-export const fonts = {
-  MERRIWEATHER: "Merriweather-Light",
-  CIRCULAR: "CircularStd",
-  SORAY: "SoRay"
-};
+import {Platform} from 'react-primitives';
```
Comment (palmfjord) [inline fonts.ts]: spaces :D can we maybe add prettier?

File: fonts.ts (hunk 1 above)
Comment (hedvigoscar) [reply fonts.ts]: we could also do `const fonts = Platform.OS === 'web' ? {...} : {...}` ?

Comment (hedvigoscar) [reply fonts.ts]: Oh, that looks cool actually, let me try that :+1:

Comment (palmfjord) [reply fonts.ts]: Good point @Iamsamwhoami , i do agree. Also i dont like ifs 😁

File: fonts.ts (hunk 2 above)
Comment (hedvigoscar) [reply fonts.ts]: I may have forgotten that prettier wasnt present in the project :(

**Pull request merged**